tests/src/test/scala/whisk/core/limits/ActionLimitsTests.scala

```diff
@@ -269,13 +269,13 @@ class ActionLimitsTests extends TestHelpers with WskTestHelpers {
   it should "be aborted when exceeding its memory limits" in withAssetCleaner(wskprops) { (wp, assetHelper) =>
     val name = "TestNodeJsMemoryExceeding"
     assetHelper.withCleaner(wsk.action, name, confirmDelete = true) {
-      val allowedMemory = 256.megabytes
+      val allowedMemory = 128.megabytes
       val actionName = TestUtils.getTestActionFilename("memoryWithGC.js")
       (action, _) =>
         action.create(name, Some(actionName), memory = Some(allowedMemory))
     }
 
-    val run = wsk.action.invoke(name, Map("payload" -> 512.toJson))
+    val run = wsk.action.invoke(name, Map("payload" -> 256.toJson))
```

Comment:
Bad push? Those changes are unrelated, no?

Reply:
Travis failed “Action limits should be aborted when exceeding its memory limits” after I increased kafka instance number to 2. (All good in my local environment & dist environment)

```
whisk.core.limits.ActionLimitsTests > Action limits should be aborted when exceeding its memory limits STANDARD_OUT
    check failed for activation 106409d09700435da409d09700835d69: {"duration":315,"name":"TestNodeJsMemoryExceeding","subject":"guest","activationId":"106409d09700435da409d09700835d69","publish":false,"annotations":[{"key":"limits","value":{"timeout":60000,"memory":256,"logs":10}},{"key":"path","value":"guest/TestNodeJsMemoryExceeding"},{"key":"kind","value":"nodejs:6"}],"version":"0.0.1","response":{"result":{"error":"An error has occurred: RangeError: Array buffer allocation failed"},"success":false,"status":"action developer error"},"end":1511323308912,"logs":["2017-11-22T04:01:48.83040658Z  stdout: helloEatMemory memory 512MB","2017-11-22T04:01:48.899278892Z stderr: RangeError: Array buffer allocation failed","2017-11-22T04:01:48.899342652Z stderr: at Buffer.Uint8Array (native)","2017-11-22T04:01:48.899350793Z stderr: at FastBuffer (buffer.js:11:5)","2017-11-22T04:01:48.89935681Z  stderr: at createUnsafeBuffer (buffer.js:38:12)","2017-11-22T04:01:48.899362782Z stderr: at new Buffer.alloc (buffer.js:130:12)","2017-11-22T04:01:48.899368458Z stderr: at eat (eval at <anonymous> (/nodejsAction/runner.js:76:109), <anonymous>:3:18)","2017-11-22T04:01:48.899379285Z stderr: at NodeActionRunner.main [as userScriptMain] (eval at <anonymous> (/nodejsAction/runner.js:76:109), <anonymous>:11:5)","2017-11-22T04:01:48.899386376Z stderr: at /nodejsAction/runner.js:95:45","2017-11-22T04:01:48.899391497Z stderr: at NodeActionRunner.run (/nodejsAction/runner.js:89:16)","2017-11-22T04:01:48.899397412Z stderr: at doRun (/nodejsAction/src/service.js:155:31)","2017-11-22T04:01:48.899402993Z stderr: at runCode (/nodejsAction/src/service.js:111:20)"],"start":1511323308597,"namespace":"guest"}
    Exception occurred during test execution: org.scalatest.exceptions.TestFailedException: "An error has occurred: RangeError: Array buffer allocation failed" was not equal to "The action exhausted its memory and was aborted."
```

I doubt that travis suffers low free memory problem caused by memory consumption of additional kafka&zookeeper instance. I also reduced kafka default heap size to 0.5g with same reason. 
But it had been changed before I changed kafka image, so I try rollback and run Travis again.